agent/lib/utils.js

```diff
@@ -286,3 +286,26 @@ module.exports.get = function (object, fields, default_value) {
     }
     return o || default_value;
 };
+
+var multipliers = {};
+multipliers.B = 1;
+multipliers.KB = ( multipliers.B * 1024 );
+multipliers.MB = ( multipliers.KB * 1024 );
+multipliers.GB = ( multipliers.MB * 1024 );
+module.exports.parseToBytes = function (input) {
+    var unit = input.toUpperCase().match(/([KMGTPEZ]I?)?B$/);
+
+    if (!unit) {
+        log.warn( "parseToBytes input does not contain a supported unit of measurement.  Using 'B': %s", input );
```

Comment:
We should probably use 'k' or both 'k' and 'K', because 'k' is the official.

Reply:
by switching it to upper case(toUpperCase) both will be allowed.

Reply:
Ah, i didn't catch that